public/app/features/expressions/components/Threshold.tsx

```diff
@@ -85,11 +86,13 @@ export const Threshold = ({ labelWidth, onChange, refIds, query, onError, useHys
 
   const hysteresisEnabled = Boolean(config.featureToggles?.recoveryThreshold) && useHysteresis;
 
+  const id = uniqueId('threshold-');
+
   return (
     <>
       <InlineFieldRow>
-        <InlineField label="Input" labelWidth={labelWidth}>
-          <Select onChange={onRefIdChange} options={refIds} value={query.expression} width={20} />
+        <InlineField label="Input" labelWidth={labelWidth} htmlFor={id}>
+          <Select inputId={id} onChange={onRefIdChange} options={refIds} value={query.expression} width={20} />
```

Comment:
Just adding IDs to link the two together in an accessible way. I was using this in a test to select the node for threshold, but I had to skip that test - but this change makes sense anyway IMO